src/app/perspective-status.component.html

```diff
@@ -1,25 +1,45 @@
-<div id="widgetContainer" class="horizontal-container">
+<div #widgetContainer id="widgetContainer" class="horizontal-container">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
         rel="stylesheet">
   <!-- For items that need to be animated, hidden is used instead of *ngIf.
        This is because *ngIf removes items from the DOM, which means they
        cannot be found when the component is initializing if their display
        conditions are not met. -->
   <div id="animationContainer">
-    <button
-         id="statusWidget"
-         class="dot"
-         [style.width]="indicatorWidth"
-         [style.height]="indicatorHeight"
-         [hidden]="shouldHideStatusWidget"
-         [attr.aria-hidden]="shouldHideStatusWidget || isPlayingShowOrHideLoadingWidgetAnimation"
-         [disabled]="isPlayingShowOrHideLoadingWidgetAnimation"
-         role="button"
-         [attr.aria-label]=
-            "getAnimationA11yLabel(showScore, isPlayingLoadingAnimation, isPlayingShowOrHideDetailsAnimation)"
-         (click)="toggleScore()"
-         tabindex=0>
-    </button>
+    <div *ngIf="loadingIconStyle === loadingIconStyleConst.DEFAULT"
+         id="defaultLoadingIconContainer">
+      <button
+           #defaultStatusWidget
+           id="defaultStatusWidget"
+           class="dot"
+           [style.width]="indicatorWidth"
+           [style.height]="indicatorHeight"
+           [hidden]="shouldHideStatusWidget || alwaysHideLoadingIcon"
+           [attr.aria-hidden]="shouldHideStatusWidget || isPlayingShowOrHideLoadingWidgetAnimation || alwaysHideLoadingIcon"
```

Comment:
Yeah, this is a problem throughout the code: loadingIcon == loadingWidget == statusWidget == widget. I need to do another pass and clean this up, since it affects a lot more variables than just these ones. I'm ambivalent about whether to make that naming change here or in a follow-up change. What is your preference?

Reply:
In a followup is fine.